composer.json

```diff
@@ -55,7 +56,7 @@
             "dev-master": "3.x-dev"
         },
         "symfony": {
-            "require": "4.4.*"
+            "require": "4.4.*|5.0.*"
```

Comment:
This was removed on master so after you rebase this should be deleted.

Reply:
yes, it removed it after merge from master, thank you!